artemis-server/src/main/java/org/apache/activemq/artemis/core/settings/impl/AddressSettings.java

```diff
@@ -1192,6 +1212,8 @@ public void encode(ActiveMQBuffer buffer) {
       BufferHelper.writeNullableBoolean(buffer, defaultLastValueQueue);
 
       BufferHelper.writeNullableLong(buffer, redistributionDelay);
+      
+      BufferHelper.writeNullableBoolean(buffer, clusteredQueues);
```

Comment:
this cannot be added here, new bits in the journal must ALWAYS be at the end for compatibility reasons